README.md

```diff
@@ -141,17 +141,19 @@ The STL uses boost-math headers to provide P0226R1 Mathematical Special Function
 acquire this dependency.
 
 1. Install Visual Studio 2019 16.6 Preview 2 or later.
-2. Invoke `git clone https://github.com/microsoft/vcpkg`
-3. Invoke `cd vcpkg`
-4. Invoke `.\bootstrap-vcpkg.bat`
-5. Assuming you are targeting x86 and x64, invoke `.\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows`
+2. Open a command prompt and change directories to a location where you'd like a clone of this STL repository.
+3. Invoke `git clone https://github.com/microsoft/STL`
+4. Invoke `cd STL`
+5. Invoke `git submodule update --init --progress vcpkg`
+6. Invoke `cd vcpkg`. Alternatively, you can use a separate installation of vcpkg.
+7. Invoke `.\bootstrap-vcpkg.bat`
+8. Assuming you are targeting x86 and x64, invoke `.\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows`
    to install the boost-math dependency. Add `boost-math:arm-windows boost-math:arm64-windows` to this to target ARM
    and ARM64.
-6. Run `.\vcpkg.exe integrate install` which tells Visual Studio which vcpkg instance you wish to use. If you have never
+9. Run `.\vcpkg.exe integrate install` which tells Visual Studio which vcpkg instance you wish to use. If you have never
    done this before, you may be prompted to elevate.

```

Comment:
Is this actually a good idea? This will set the global vcpkg install to be the one in the repo and might break people's setup, no?

Reply:
We don't actually need users to run the integrate command because we don't need the vcpkg include directory to magically be discovered by VS. We can just have users set their CMake toolchain file to be the vcpkg.cmake file in the submodule.